rest-api/src/main/resources/paths/index.yml

```diff
@@ -276,6 +276,8 @@
     $ref: ./participants/v3_participants_healthCode.yml
 /v3/participants/syn:{synapseUserId}:
     $ref: ./participants/v3_participants_synapseUserId.yml
+/v3/participants/{userId}/enrollments:
+    $ref: ./participants/v3_participants_userId_enrollments.yml
```

Comment:
This was just missing from the SDK despite the fact that it exists on the server, which I discovered while trying to change it's paging type.